docs/specifications/tests/Connectivity-TP/connectivity01.md

```diff
@@ -1,39 +1,218 @@
-the domain are accessible over UDP on port 53
 
-### Inputs
+## Objective
 
-The domain to be tested.
+UDP is the fundamental protocol to reach a general purpose name server hosting
+a zone, "DNS servers MUST be able to service UDP [...]" ([RFC 1123], section
+6.1.3.2, page 75), also restated in [RFC 7766][RFC 7766#5], section 5.
 
-### Ordered description of steps to be taken to execute the test case
+This test case will verify if the name servers in the delegation of and zone of
+*Child Zone* are reachable over UDP.
 
-1. Obtains the IP address of the name servers from [Method4](../Methods.md)
-   and [Method5](../Methods.md)
-2. A SOA query is sent over UDP to distinct IP addresses of each name server
-   found in step 1.
-3. If all queries in step 2 receive a DNS answer (bogus responses are not
-   checked here) then the test case succeed.
+Most Zonemaster test cases will query the name servers in the delegation or the
+name servers appointed by the NS records in the zone for the NS or SOA record,
+or both. It is crucial that problems are reported, but instead of letting several
+test cases report the same problems found, most test cases assume that this test
+case is run. Only this test case will report problems found in the following
+areas over UDP:
 
-### Outcome(s)
+* Name Server not responding to a query without EDNS.
+* Name Server not including SOA record of *Child Zone* in the answer section in
+  the response on a SOA query for *Child Zone*.
+* Name Server not including NS record of *Child Zone* in the answer section in
+  the response on an NS query for *Child Zone*.
+* Name Server not setting the AA flag in a response with SOA or NS in answer
+  section.
+* Name Server responding with unexpected [RCODE Name] (any except "NoError") on
+  query for SOA or NS for *Child Zone*.
 
-If there is any name server that fails to answer queries over port 53 using
-UDP, this test case fails
 
-### Special procedural requirements	
+## Scope
 
-If either IPv4 or IPv6 transport is disabled, ignore the evaluation of the result of any test using this transport protocol. Log a message reporting on the ignored result.
+The only UDP port defined for DNS is port 53 ([RFC 1035][RFC 1035#4.2.1], section
+4.2.1), and that is the only port used by this and other test cases for DNS
+queries to the name servers.
+
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+
+## Summary
+
+Message Tag                         |Level  |Arguments  | Message ID for message tag
+:-----------------------------------|:------|:----------|:------------------------------------------------------------------------------------------
+CN01_IPV4_DISABLED                  | NOTICE| ns_list   | IPv4 is disabled. No DNS queries are sent to these name servers: "{ns_list}".
+CN01_IPV6_DISABLED                  | NOTICE| ns_list   | IPv6 is disabled. No DNS queries are sent to these name servers: "{ns_list}".
```

Comment:
> NOTICE

Remove trailing space

Reply:
Trailing space where?

Reply:
Before "NOTICE". Or rather, you can expand the table a bit to put spaces before and after in the "Level" column (just like in the other columns)

Reply:
Nevermind this, it looks good when page is rendered.

Reply:
Yes, white space before or after the text does not affect rendering, at least not in Github. I wanted to keep the lines as short as possible.